examples/build.gradle

```diff
@@ -21,30 +21,29 @@ repositories {
   mavenLocal()
 }
 
-apply plugin: 'kotlin'
-apply plugin: 'com.google.protobuf'
+apply plugin: "kotlin"
+apply plugin: "com.google.protobuf"
 
 // Generate IntelliJ IDEA's .idea & .iml project files.
 // protobuf-gradle-plugin automatically registers *.proto and the gen output files
 // to IntelliJ as sources.
 // For best results, install the Protobuf and Kotlin plugins for IntelliJ.
-apply plugin: 'idea'
-apply plugin: 'application'
+apply plugin: "idea"
+apply plugin: "application"
 
 dependencies {
   // This repo
-  implementation "io.grpc:grpc-kotlin-stub:${grpc_kotlin_version}"
+  implementation "io.grpc:grpc-kotlin-stub:$grpc_kotlin_version"
 
-  implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.61"
-  implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:${coroutines_version}"
+  implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
```

Comment:
No version for `kotlin:kotlin-stdlib-jdk8`?

Reply:
The plugin fills it in based on the Kotlin version.

Reply:
Might it make sense to add a comment to that effect, or is it too much common knowledge?

Reply:
The official docs for Kotlin use this convention and it's common practice. So I think we should be good.